src/main/java/org/opentripplanner/routing/api/request/ViaLocationDeprecated.java

```diff
@@ -11,8 +11,12 @@
  * @param passThroughPoint Does the via point represent a pass through
  * @param minSlack         Minimum time that is allowed to wait for interchange.
  * @param maxSlack         Maximum time to wait for next departure.
+ *
+ * @deprecated We will replace the complex via-search with a simpler version part of the
+ *      existing trip search.
```

Comment:
I don't know if we should refer to an APIs "trip" query in the core code since trip can mean two things.